examples/api/test/widgets/animated_list/animated_list.0_test.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/animated_list/animated_list.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets(
+    'Items can be selected, added, and removed from AnimatedList',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(const example.AnimatedListSample());
+
+      expect(find.text('Item 0'), findsOne);
+      expect(find.text('Item 1'), findsOne);
+      expect(find.text('Item 2'), findsOne);
+
+      // Add an item at the end of the list
```

Comment:
```suggestion
      // Add an item at the end of the list.
```

Reply:
I've updated them in [test: Add test for animated list example](https://github.com/flutter/flutter/pull/156452/commits/3668cd030b801f2d1491e0b191ca1a4ba60b5978) :)